### ISSUE TITLE
Tarball download fails via Flathub in KDE/Discover

### ISSUE THREAD

**Issue by ag-gaphp** [flathub/us.zoom.Zoom#479]
Error received when installing via Flathub in KDE Discover:

```
Aborted due to failure (While downloading https://cdn.zoom.us/prod/6.1.11.1545/zoom_x86_64.tar.xz: While fetching https://cdn.zoom.us/prod/6.1.11.1545/zoom_x86_64.tar.xz: [56] Failure when receiving data from the peer)
```

When I try to curl the URL in this error, the connection is reset. Firefox also fails to find the remote file.

~~Looking further into it, when I choose "Other Linux OS" on [this download page for Zoom](https://zoom.us/download), the URL should be: `https://zoom.us/client/6.1.11.1545/zoom_x86_64.tar.xz`~~

**Comment by takluyver** [comment]
The original cdn.zoom.us URL seems to work for me in curl or in Firefox. Maybe it was just temporarily down. Can you try again?

**Comment by ag-gaphp** [comment]
Odd. I tried again a few hours later after I posted this on the same machine and it was still failing.

I've just tried again on a different machine and different network, but the CDN URL still can't be reached from Firefox or Discover/flatpak cli, but it does download using curl until ~29% and then the connection gets reset by the peer. Fairly consistently it fails at 29% after a few attempts.

Not sure what the deal is. I've been having similar issues with other packages, and so far just been using AURs instead without issues. Must be something with my config as all these machines I'm testing are fresh installs of Arch using the same setup script.
